.azure-pipelines/pipelines.yml

```diff
@@ -571,6 +571,42 @@ stages:
         AZP_SHA1: $(Build.SourceVersion)
         DOCKERHUB_USERNAME: $(DockerUsername)
         DOCKERHUB_PASSWORD: $(DockerPassword)
+        WINDOWS_BUILD_TYPE: "windows"
+        WINDOWS_IMAGE_BASE: "mcr.microsoft.com/windows/servercore"
+        WINDOWS_IMAGE_TAG: "ltsc2019"
+    - task: PublishBuildArtifacts@1
+      inputs:
+        pathtoPublish: "$(Build.StagingDirectory)/build_images"
+        artifactName: docker_windows
+      condition: always()
+  - job: dockerWindowsServer2022
```

Comment:
If we are distinguishing WindowsServer and WindowsNanoServer, shouldn't this become part of the job name? Isn't it reasonable to expect that both for 2022 might be interesting to one user or another, until we have distroless to let the user choose?

Reply:
> If we are distinguishing WindowsServer and WindowsNanoServer, shouldn't this become part of the job name? 

Yes I can change that

> Isn't it reasonable to expect that both for 2022 might be interesting to one user or another, until we have distroless to let the user choose?

I dont see the motivation for servercore given the size and that nanoserver is the blessed way forward. I am having a lazy approach where we could add servercore images if someone asks for it. I kept the servercore based images for 2019 only because we already publish it

Reply:
fixed